java/src/apps/GuiLafConfigPane.java

```diff
@@ -97,7 +97,7 @@ void doClickSelection(JPanel panel) {
     void doVerticalToolBar(JPanel panel) {
         panel.setLayout(new FlowLayout());
         mouseEvent = new JCheckBox(ConfigBundle.getMessage("GUIButtonVerticalToolBar"));
-        mouseEvent.setSelected(InstanceManager.getDefault(GuiLafPreferencesManager.class).isVerticalToolBar());
+        mouseEvent.setSelected(InstanceManager.getDefault(GuiLafPreferencesManager.class).isToolBarVertical());
```

Comment:
Is this something that should be constant across all panels within the given profile, or should this be settable per panel?